### PR TITLE
🧹 Update devtools-ton

### PR DESCRIPTION
**Pull request by KaiStevenson**

### CODE REVIEW

**Comment by KaiStevenson** [comment]
@SocketSecurity ignore npm/@layerzerolabs/lz-ton-sdk-v2@3.0.24-ton.0

File: packages/devtools-ton/jest.config.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ const config: Config = {
...
     reporters: [['github-actions', { silent: false }], 'default'],
     testEnvironment: 'node',
-    testTimeout: 60_000,
     moduleNameMapper: {
         '^@/(.*)$': '<rootDir>/src/$1',
     },
     transform: {
         '^.+\\.(t|j)sx?$': '@swc/jest',
     },
+    testTimeout: 250000,
```
Comment (yargo13) [inline packages/devtools-ton/jest.config.ts:13]: Bring this back to original value

File: packages/devtools-ton/CHANGELOG.md [hunk 2]
```diff
@@ -7,3 +7,11 @@
...
 - Updated dependencies [1bb0524]
   - @layerzerolabs/devtools@0.4.2
+
+## 0.0.3
+
+### Patch Changes
+
+- Upgraded dependency (@layerzerolabs/lz-definitions 3.0.21->3.0.27)
+- Implemented new SDK methods
+- Added new tests
```
Comment (yargo13) [inline packages/devtools-ton/CHANGELOG.md:17]: Don't need to do it manually, changeset will do it

File: packages/devtools-ton/package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@layerzerolabs/devtools-ton",
-  "version": "0.0.2",
+  "version": "0.0.3",
+  "private": true,
```
Comment (yargo13) [inline packages/devtools-ton/package.json:4]: Don't need to update manually, and shouldn't be private

File: packages/devtools-ton/package.json [hunk 4]
```diff
@@ -31,33 +32,37 @@
...
-    "@layerzerolabs/lz-definitions": "^3.0.21",
+    "@layerzerolabs/io-devtools": "~0.1.14",
+    "@layerzerolabs/lz-definitions": "^3.0.27",
     "@ton/core": "^0.59.0",
     "@ton/crypto": "^3.3.0",
     "@ton/ton": "^15.1.0"
   },
   "publishConfig": {
-    "access": "public"
+    "access": "restricted"
```
Comment (yargo13) [inline packages/devtools-ton/package.json:66]: This can be public

File: packages/devtools-ton/package.json [hunk 5]
```diff
@@ -31,33 +32,37 @@
...
     "tslib": "~2.6.2",
     "tsup": "~8.0.1",
-    "typescript": "^5.4.4"
+    "typescript": "^5.3.3"
   },
   "peerDependencies": {
     "@layerzerolabs/devtools": "~0.4.2",
-    "@layerzerolabs/io-devtools": "~0.1.13",
-    "@layerzerolabs/lz-definitions": "^3.0.21",
+    "@layerzerolabs/io-devtools": "~0.1.14",
```
Comment (yargo13) [inline packages/devtools-ton/package.json:59]: Check that all other places have this updated as well

**Review by yargo13 — CHANGES_REQUESTED** [review]

**Review by yargo13 — APPROVED** [review]

**Pull request merged**